test/test_utils.py

```diff
@@ -0,0 +1,23 @@
+import functools
+
+import peewee
+
+db = peewee.SqliteDatabase(':memory:')
```

Comment:
This might be a fixture

Reply:
As it has been used in a decorator and only one insatance in needed and also I do not think fixtures will be passed to decorators marked on test methods.